wagtail/contrib/search_promotions/forms.py

```diff
@@ -75,4 +140,4 @@ def clean(self):
             non_deleted_forms < self.minimum_forms
             or non_empty_forms < self.minimum_forms
         ):
-            raise forms.ValidationError(self.minimum_forms_message)
+            raise forms.ValidationError(self.MINIMUM_FORMS_MESSAGE)
```

Comment:
```suggestion
            raise forms.ValidationError(self.minimum_forms_message)
```

As per comments above, please keep this code as is, no need to change as part of this PR.

If you want to update this later, that's best suited as a standalone PR.